packages/core/strapi/lib/commands/routes/list.js

```diff
@@ -3,11 +3,25 @@
 const CLITable = require('cli-table3');
 const chalk = require('chalk');
 const { toUpper } = require('lodash/fp');
+const tsUtils = require('@strapi/typescript-utils');
 
 const strapi = require('../../index');
 
 module.exports = async function() {
-  const app = await strapi().load();
+  const appDir = process.cwd();
```

Comment:
I'm not gonna repeat it but I guess you will have to review all the commands now :D 

I think this really needs to be calling a  build function without much more knowledge instead of repeat all of this

```js

const ctx = strapi.compile(appDir);

const app = await strapi(ctx);
```

Bascially what I meant with internal toolchain

Reply:
But then we link @strapi/utils to @strapi/strapi's core, would that still be ok?